test/unit/region.spec.js

```diff
@@ -1031,6 +1035,10 @@ describe('region', function() {
   describe('when getting a region', function() {
     beforeEach(function() {
       this.itemView = new Backbone.Marionette.View();
+      this.itemView.template = function() {
+        return 'content';
+      };
+      this.itemView.render = sinon.stub();
```

Comment:
there's a sinon attached to `this` that auto cleans up, so we tend to use it `this.sinon.stub()`